### DISCUSSION TITLE
Error with orc8r_test container on Development Environement

### DISCUSSION THREAD

**Discussion by shalinjshah18** [magma/magma#7076] [Q&A] [answered]
Hi Team I am setting up development environment on Debian Virtual Machine using quick starter guide available.

While starting all docker containers machine using sudo ./run.py --metrics I do not see orc8r_test container coming up.

when I see docker image logs I find below error

admin@VM211:~/magma/orc8r/cloud/docker$ sudo docker logs f4a7234076be
mesg: ttyname failed: Inappropriate ioctl for device
Hello World!
mesg: ttyname failed: Inappropriate ioctl for device
Hello World!
mesg: ttyname failed: Inappropriate ioctl for device
Hello World!
mesg: ttyname failed: Inappropriate ioctl for device
Hello World!

Need help in resolving this error.

**Comment by amarpad** [comment, marked as answer]
This is not related to magma at all, and is more a setup issue pertaining to docker.

**Comment by shalinjshah18** [comment]
Hi Amar, 
Thanks for response, however I did install complete fresh Mac OS v10.15.7 and re-run setup again for development evn.
Steps to build docker images were all successful and I was able to get all containers up now including fluentd but only orc8r_test fails with same error. It would be of great help if you can please share what this particular error is pertaining docker and will take it from there.
